source/gameengine/Converter/KX_ConvertActuators.cpp

```diff
@@ -977,7 +977,7 @@ void BL_ConvertActuators(const char* maggiename,
 			}
 
 			tmp = new SCA_2DFilterActuator(gameobj, filtermode,  _2dfilter->flag,
-			                               _2dfilter->float_arg, _2dfilter->int_arg,
+			                               _2dfilter->float_arg, _2dfilter->int_arg, _2dfilter->mipmap,
```

Comment:
@youle31: if you set the constructor argument type of mipmap to a bool, the int value from _2dfilter->mipmap will be casted to it implicitly, if you compiler complain to an explicit cast: `bool(value)`.